bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/types/BFunctionType.java

```diff
@@ -169,7 +169,8 @@ public String toString() {
         if (SymbolFlags.isFlagOn(this.flags, SymbolFlags.ANY_FUNCTION)) {
             stringRep = "function";
         } else {
-            stringRep = "function (" + (parameters != null ? getTypeListAsString(parameters) : "") + ")" +
+            stringRep = "function (" + (parameters != null ? getTypeListAsString(parameters) : "") +
+                    (restType != null ? "," + ((BArrayType) restType).getElementType().toString() + "..." : "") + ")" +
```

Comment:
Shall we check that casting the `restType` into `BArrayType` will not cause a CCE?
If we always have the `restType` as an `ArrayType`, we can change the type of that field.

Reply:
I checked this by building the ballerina-lang with tests and found out restType will be either `null` or instanceOf `BArrayType`.